spring-cloud-dataflow-server-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/cloudfoundry/CloudFoundryDataFlowServer.java

```diff
@@ -26,7 +28,7 @@
  *
  * @author Eric Bottard
  */
-@SpringBootApplication(exclude = SessionAutoConfiguration.class)
+@SpringBootApplication(exclude = {SessionAutoConfiguration.class, RedisAutoConfiguration.class, DataSourceAutoConfiguration.class})
```

Comment:
Disabling these auto-config would break analytics in standalone mode(running CF server outside CF). Also server doesn't even start outside of a CF
```
$ java -jar spring-cloud-dataflow-server-cloudfoundry/target/spring-cloud-dataflow-server-cloudfoundry-1.6.0.BUILD-SNAPSHOT.jar
***************************

Description:

Parameter 0 of constructor in org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration required a bean of type 'javax.sql.DataSource' that could not be found.
	- Bean method 'dataSource' not loaded because @ConditionalOnProperty (spring.datasource.jndi-name) did not find property 'jndi-name'
	- Bean method 'dataSource' not loaded because @ConditionalOnBean (types: org.springframework.boot.jta.XADataSourceWrapper; SearchStrategy: all) did not find any beans


Action:

Consider revisiting the conditions above or defining a bean of type 'javax.sql.DataSource' in your configuration.

```

Reply:
Also we can't just disable `DataSourceAutoConfiguration` as it does more than just adding `dataSource` bean.

Reply:
Also when disabling `RedisAutoConfiguration`, we miss other things it does.

Reply:
I didn't consider stand alone mode, we are going to have to obviously fix this.
I believe all RedisAutoConfiguration does in addition is create the templates, but yea - you are right with DataSourceAutoConfiguration.   With the additional return of a `RedisConnectionFactory redisFactory() ` now, maybe it will work w/o turning off autoconfiguration.  The whole process has been hit or miss as the root cause is not known yet.

Reply:
that seemed to do the trick.  updating the PR.